deployer/templates/heapster.yaml

```diff
@@ -83,9 +82,9 @@ objects:
           - "--wrapper.username_file=/hawkular-account/hawkular-metrics.username"
           - "--wrapper.password_file=/hawkular-account/hawkular-metrics.password"
           - "--wrapper.allowed_users_file=/secrets/heapster.allowed-users"
-          - "--wrapper.endpoint_check=https://hawkular-metrics:443/hawkular/metrics/status"
+          - "--wrapper.endpoint_check=https://hawkular-metrics.openshift-infra.svc.cluster.local:443/hawkular/metrics/status"
```

Comment:
Changing it back isn't an option, as the name on the cert is a FQDN, not just `hawkular-metrics` (IIRC). But I can certainly make it use the API. I just need to figure out which API you mean :)